arrows/serialize/protobuf/tests/test_convert_protobuf.cxx

```diff
@@ -516,56 +516,32 @@ TEST( convert_protobuf, geo_polygon )
 // ----------------------------------------------------------------------------
 TEST( convert_protobuf, metadata )
 {
-  static kwiver::vital::metadata_traits traits;
   kwiver::vital::metadata meta;
+  meta.add< kwiver::vital::VITAL_META_METADATA_ORIGIN >( "test-source" );
+  meta.add< kwiver::vital::VITAL_META_UNIX_TIMESTAMP >( 12345678 );
+  meta.add< kwiver::vital::VITAL_META_SENSOR_VERTICAL_FOV >( 12345.678 );
 
   {
-    const auto& info = traits.find( kwiver::vital::VITAL_META_METADATA_ORIGIN );
-    auto item = info.create_metadata_item( kwiver::vital::any(std::string ("test-source")) );
-    meta.add( std::move( item ) );
+    kwiver::vital::geo_point::geo_2d_point_t geo_2d{ 42.50, 73.54 };
+    kwiver::vital::geo_point pt{ geo_2d, kwiver::vital::SRID::lat_lon_WGS84 };
+    meta.add< kwiver::vital::VITAL_META_FRAME_CENTER >( pt );
   }
 
   {
-    const auto& info = traits.find( kwiver::vital::VITAL_META_UNIX_TIMESTAMP );
-    auto item = info.create_metadata_item( kwiver::vital::any((uint64_t)12345678) );
-    meta.add( std::move( item ) );
+    kwiver::vital::geo_point::geo_3d_point_t geo{ 42.50, 73.54, 16.33 };
```

Comment:
The change here seems to replace a TIMESTAMP test with a geographical point test.

Reply:
See my reply to similar comment above.